### PR TITLE
add more upgrade periodic jobs

### PR DESCRIPTION
**Pull request by carolynhu**
https://github.com/istio/istio/issues/17950

### CODE REVIEW

File: prow/cluster/jobs/all-periodics.yaml [hunk 1]
```diff
@@ -279,8 +279,58 @@ periodics:
           - entrypoint
           - prow/perf_benchmark_test.sh
 
-- cron: "0 18 * * *"  # starts every day at 18:00PM UTC
-  name: istio-upgrade-helm-chart
+# upgrade using helm from 1.2.8 to 1.3.4
+- cron: "0 2 * * *"  # starts every day at 02:00AM UTC
+  name: istio-upgrade-using-helm
```
Comment (howardjohn) [inline prow/cluster/jobs/all-periodics.yaml]: job names need to be unique, should probably include from and to in the name

File: prow/cluster/jobs/all-periodics.yaml [hunk 2]
```diff
@@ -279,8 +279,58 @@ periodics:
...
+    testgrid-dashboards: istio_release-pipeline
+    testgrid-alert-email: istio-oncall@googlegroups.com
+    testgrid-num-failures-to-alert: '1'
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+      - <<: *istio_container_with_kind
+        env:
+          - name: SOURCE_TAG
```
Comment (howardjohn) [inline prow/cluster/jobs/all-periodics.yaml:302]: I think we should just always set TARGET_TAG, to be more explicit

**Review by howardjohn — COMMENTED** [review]
lgtm other than the names

File: prow/cluster/jobs/all-periodics.yaml (hunk 2 above)
Comment (carolynhu) [reply prow/cluster/jobs/all-periodics.yaml:302]: ok, I see, I will add TARGET_TAG for every job

File: prow/cluster/jobs/all-periodics.yaml (hunk 1 above)
Comment (carolynhu) [reply prow/cluster/jobs/all-periodics.yaml]: oh, I see.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**